fdbserver/MockGlobalState.actor.cpp

```diff
@@ -723,7 +723,7 @@ bool MockGlobalState::serverIsDestForShard(const UID& serverId, KeyRangeRef shar
 }
 
 bool MockGlobalState::allShardsRemovedFromServer(const UID& serverId) {
-	return allServers.count(serverId) && shardMapping->getNumberOfShards(serverId) == 0;
+	return !allServers.contains(serverId) && shardMapping->getNumberOfShards(serverId);
```

Comment:
```suggestion
	return allServers.contains(serverId) && shardMapping->getNumberOfShards(serverId) == 0;
```

Reply:
Thanks! And good catch, updated the PR with the fix. Also re-running 100K Joshua.